client/src/components/profile.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import {
+    Form,
+    Button,
+    Badge
+} from 'react-bootstrap';
+import axios from "axios";
+import "../App.css";
+import profilepic from "./maxresdefault.jpg"
+
+class RenderProfilePage extends Component{
+    
+    /*
```

Comment:
Are these comments necessary?

Reply:
I just commented it out for now because it is not finished, I wanted to push something to show that there is some work that is being done

Reply:
Good idea